### ISSUE TITLE
Auto-Tagging issues with 4-digit issue numbers fails

### ISSUE THREAD

**Issue by Nebelherr23** [comictagger/comictagger#496]
***Runtime Info***
*OS* - Arch Linux
*Python* - 3.11.3
*CT Version* - 1.5.5

(reproduced on Windows 10 and 11)

*** What's the issue? ***
CT fails to automatically identify issues with issue numbers > 3 digits. It identifies the series but claims there's no matching issue.

*** How to reproduce ***

- Load up a comic issue with 4-digit issue number (Detective or Action Comics spring to mind).
- Auto-Identify / Auto-Tag issue

**Comment by Nebelherr23** [comment]
Now this is embarrassing...

It seems that it is caused by a weird interaction between the complicated parser and my naming scheme... switching off the parser setting has it working -.-'

**Comment by lordwelch** [comment]
@Nebelherr23 Can you provide an example? I can add it to the automated tests

**Comment by Nebelherr23** [comment]
Basically as I said: Anything with 4-digit issue numbers. Current run of Action Comics or Detective Comics is a decent example

```'Action Comics 1000 (2018).cbz'
'Action Comics 1001 (2018).cbz'
'Action Comics 1002 (2018).cbz'
'Action Comics 1003 (2018).cbz'
'Action Comics 1004 (2018).cbz'
'Action Comics 1005 (2019).cbz'
'Action Comics 1006 (2019).cbz'
'Action Comics 1007 (2019).cbz'
'Action Comics 1008 (2019).cbz'
'Action Comics 1009 (2019).cbz'
'Action Comics 1010 (2019).cbz'
'Action Comics 1011 (2019).cbz'
'Action Comics 1012 (2019).cbz'
'Action Comics 1013 (2019).cbz'
'Action Comics 1014 (2019).cbz'
'Action Comics 1015 (2019).cbz'
'Action Comics 1016 (2019).cbz'
'Action Comics 1017 (2020).cbz'
'Action Comics 1018 (2020).cbz'
'Action Comics 1019 (2020).cbz'
'Action Comics 1020 (2020).cbz'
'Action Comics 1021 (2020).cbz'
'Action Comics 1022 (2020).cbz'
'Action Comics 1023 (2020).cbz'
'Action Comics 1024 (2020).cbz'
'Action Comics 1025 (2020).cbz'
'Action Comics 1026 (2020).cbz'
'Action Comics 1027 (2021).cbz'
'Action Comics 1028 (2021).cbz'
'Action Comics 1029 (2021).cbz'
'Action Comics 1030 (2021).cbz'
'Action Comics 1031 (2021).cbz'
'Action Comics 1032 (2021).cbz'
'Action Comics 1033 (2021).cbz'
'Action Comics 1034 (2021).cbz'
'Action Comics 1035 (2021).cbz'
'Action Comics 1036 (2021).cbz'
'Action Comics 1037 (2022).cbz'
'Action Comics 1038 (2022).cbz'
'Action Comics 1039 (2022).cbz'
'Action Comics 1040 (2022).cbz'
'Action Comics 1041 (2022).cbz'
'Action Comics 1042 (2022).cbz'
'Action Comics 1043 (2022).cbz'
'Action Comics 1044 (2022).cbz'
'Action Comics 1045 (2022).cbz'
'Action Comics 1046 (2022).cbz'
'Action Comics 1047 (2022).cbz'
'Action Comics 1048 (2022).cbz'
'Action Comics 1049 (2023).cbz'
'Action Comics 1050 (2023).cbz'
'Action Comics 1051 (2023).cbz'
'Action Comics 1052 (2023).cbz'
'Action Comics 1053 (2023).cbz'
'Action Comics 1054 (2023).cbz'
'Action Comics 1055 (2023).cbz'
'Action Comics 1056 (2023).cbz'
```

**Comment by lordwelch** [comment]
I fixed this in [1.6.0-alpha.1]

[1.6.0-alpha.1]: https://github.com/comictagger/comictagger/releases/tag/
